resources/packs/azure/azure_rm_network.go

```diff
@@ -293,6 +295,100 @@ func (a *mqlAzureNetwork) GetWatchers() ([]interface{}, error) {
 	return res, nil
 }
 
+func (a *mqlAzureNetworkWatcher) GetFlowLogs() ([]interface{}, error) {
+	at, err := azureTransport(a.MotorRuntime.Motor.Provider)
+	if err != nil {
+		return nil, err
+	}
+
+	ctx := context.Background()
+	token, err := at.GetTokenCredential()
+	if err != nil {
+		return nil, err
+	}
+
+	id, err := a.id()
+	if err != nil {
+		return nil, err
+	}
+	watcherName, err := a.Name()
+	if err != nil {
+		return nil, err
+	}
+
+	resourceID, err := azure.ParseResourceID(id)
+	if err != nil {
+		return nil, err
+	}
+	client, err := network.NewFlowLogsClient(at.SubscriptionID(), token, &arm.ClientOptions{})
+	if err != nil {
+		return nil, err
+	}
+	pager := client.NewListPager(resourceID.ResourceGroup, watcherName, &network.FlowLogsClientListOptions{})
+	res := []interface{}{}
+	for pager.More() {
+		page, err := pager.NextPage(ctx)
+		if err != nil {
+			return nil, err
+		}
+		for _, flowLog := range page.Value {
+			retentionPolicy, err := a.MotorRuntime.CreateResource("azure.network.watcher.flowlog.retentionpolicy",
```

Comment:
Maybe we can use an dict approach similar to https://github.com/mondoohq/cnquery/pull/716/files#diff-3dfd9c3d602d8558fc6c3a3f5783721f3c8194755b1907a589b32a9fe1457dbcR48